java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S2761_java.html

```diff
@@ -0,0 +1,27 @@
+<p>The needless repetition of an operator is usually a typo. There is no reason to write <code>!!!i</code> when <code>!i</code> will do.</p>
+<p>On the other hand, the repetition of increment and decrement operators may have been done on purpose, but doing so obfuscates the meaning, and
+should be simplified.</p>
+<p>This rule raises an issue for sequences of: <code>!</code>, <code>~</code>, <code>-</code>, and <code>+</code>.</p>
+<h2>Noncompliant Code Example</h2>
+<pre>
+int i = 1;
+
+int j = - - -i;  // Noncompliant; just use -i
+int k = ~~i;     // Noncompliant; same as i
```

Comment:
You need to update the description file again (accordingly to what we modified).